cmd/node/main.go

```diff
@@ -181,6 +183,25 @@ func main() {
 	}()
 	go Discovering(csiNodeService, logger)
 
+	// check here for volume manager readiness
+	// input parameters are ignored by Check() function - pass empty context and health check request
+	ctx := context.Background()
+	req := &grpc_health_v1.HealthCheckRequest{}
+	for {
```

Comment:
I'm not sure that addition dead loop is the good idea. Maybe, it makes sense to restrict this loop with many attempts?

Reply:
What a good place for [`tockers`](https://gobyexample.com/tickers) or `context.withCancel`.

Reply:
Duplicate comment here as well:
_If we limit number of retries - then need to exit, but node pod restart won't solve the problem with drive manager._

Reply:
Number of retries added.